### PR TITLE
Create config check

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Add check for existing config in location of Create Team Config choice.  If file exists user is notified of existing file and prompted to continue with creation.  If user cancels the existing config will open for editing and if user chooses Create New the create process will continue as usual with the creation/merging of existing values with new config in selected location.

## Release Notes

Milestone: 2.4.0

Changelog: Added check for existing team configuration file in location during create, prompting user to continue with the create action.

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [x] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [x] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [x] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

projLevel_exists_create

projLevel_exists_cancel

globalLevel_exists_create

### CODE REVIEW

**Comment by kristinochka** [comment]
The current message is `"Team Configuration File already exists in this location\n{0}\nWould you like to continue with creating a new file?"` This makes it sounds like existing file will be lost and new one will be created, however the new functionality updates the config file. I think the message should explain to the user what this action will actually do, something like `"Would you like to update the file with empty config values?"`

**Comment by JillieBeanSim** [comment]
Thanks @kristinochka for the feedback.  I feel a full explanation in the modal message could be pretty long.  We could present a short message like `Would you like to update the file?` and have a longer explanation in the README or docs?

**Comment by kristinochka** [comment]
@JillieBeanSim Is something like `"Would you like to update the file with empty profile configurations of available extenders?"` not accurate? I don't think a user needs to know what's happening with the schema file, that's inner workings of Zowe Explorer.

**Comment by JillieBeanSim** [comment]
> @JillieBeanSim Is something like `"Would you like to add templates for profiles of available extenders to the existing config file?"` close enough? I don't think a user needs to know what's happening with the schema file, that's inner workings of Zowe Explorer or technical details the end user doesn't need to understand.
> 
> Other options: `"Would you like to update the file with configurations of available extenders?"` `"Would you like to update the file with empty profile configurations of available extenders?"`

These are very long @kristinochka, you need to remember that the message also will include `Team Configuration File already exists in this location\n{0}` where {0} is the path to the config which could also be pretty long. That's why I suggested `Would you like to continue and update the file?` and a longer explanation of what this means in the documentation.

**Comment by kristinochka** [comment]
Sure, the message can just be very simple. In general though, I don't expect users would want to go read a manual about this action. When the issue with automatically updating the schema file is fixed, this can become simplified hopefully.

**Comment by kristinochka** [comment]
I did not see issues with the intended behavior 👍 .  I do think user should not need to click a link to go outside the IDE to understand this action or be aware of a documentation site, so whatever message is put here should give a good idea of what's happening.

File: packages/zowe-explorer/src/Profiles.ts [hunk 1]
```diff
@@ -1389,6 +1397,52 @@ export class Profiles extends ProfilesCache {
...
+                const message = localize(
+                    "checkExistingConfig.createNew.message",
+                    `A Team Configuration File already exists in this location\n{0}\nWould you like to continue with creating a new file?`,
+                    file.path
+                );
+                await vscode.window
+                    .showInformationMessage(message, { modal: true }, ...[createButton])
+                    .then(async (selection) => {
+                        if (selection) {
+                            location = file.path.replace(filePath, "").replace(/[\/|\\]/g, "");
```
Comment (zFernand0) [inline packages/zowe-explorer/src/Profiles.ts:1417]: was the intention here to do a `path.basename(file.path)` ?

File: packages/zowe-explorer/src/Profiles.ts [hunk 2]
```diff
@@ -1389,6 +1397,52 @@ export class Profiles extends ProfilesCache {
...
+        let found = false;
+        let location: string;
+        const existingLayers = await this.getConfigLayers();
+        for (const file of existingLayers) {
+            if (file.path.includes(filePath)) {
+                found = true;
+                const createButton = localize("checkExistingConfig.createNew.button", "Create New");
+                const message = localize(
+                    "checkExistingConfig.createNew.message",
+                    `A Team Configuration File already exists in this location\n{0}\nWould you like to continue with creating a new file?`,
```
Comment (zFernand0) [inline packages/zowe-explorer/src/Profiles.ts:1410]: Should we mention something about the behavior of `zowe config init` ?
Right now it feels like we are about to replace the entire file. 😋

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 
Left two questions but no changes required 🥳

**Comment by JillieBeanSim** [comment]
Thanks @zFernand0 for the comments, I addressed the change and reworded the message to `A Team Configuration File already exists in this location\n{0}\nContinuing may alter the existing file, would you like to proceed?`

**Review by zFernand0 — DISMISSED** [review]
Thank you for rewording the modal box!
LGTM! 😋

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**